tst/ConfigurationTest.php

```diff
@@ -120,9 +120,15 @@ public function testHandleMissingSubKeys()
         unset($options['expire_options']['1week']);
         unset($options['expire_options']['1year']);
         unset($options['expire_options']['never']);
+        $sri_key                         = array_key_first($options['sri']);
+        $valid_sri                       = $options['sri'][$sri_key];
+        $options['sri'][$sri_key]        = ''; // empty string should get replaced with default
```

Comment:
Hmm maybe I do understand the code wrong, but where is the assert for this? Your change only adds prepare steps, it does not change what is actually asserted at the end.


Ahhhh the `$options`  at the end is modified again. Well for readability I would instead clone it and name the variable something like `$expectedOptions`. (Also some newline before would not hurt).
However, this means `some invalid SRI hash` is filtered out? That's cool, but I thought you did not want to introduce validation? Ah okay, just because it's non-empty.
Then maybe name it `invalid SRI hash should be kept` to make that clear.

Reply:
Correct interpretation, yes. The empty key gets replaced because of PHP's ini parser ignoring it and the incorrect one is preserved because we don't validate it (because that would require constantly regenerating SRIs on every request, in case the files got changed, we could then cache that to reduce the load, but then have even more complex code for that caching, etc.)

I had added the comment regarding restoring the expected values for that clarification. I think this dragged out way too long already and I'll merge it as is. If you do care about the naming of the vars, feel free to raise another PR for that. THX